src/helpers/appBar.js

```diff
@@ -265,16 +270,31 @@ const CodeLabzAppBar = () => {
               </IconButton>
             </div>
             &nbsp; &nbsp;
-            <Link to={"/login"}>
-              <Button
-                variant="contained"
-                color="primary"
-                style={{ backgroundColor: "royalblue" }}
-                data-testId="navbarlogin"
-              >
-                Log In
-              </Button>
-            </Link>
+                        {location.pathname != "/login" && location.pathname != "/signup" &&
+                        <Link to={"/login"}>
+                            <Button
+                                variant="contained"
+                                color="primary"
+                                style={{ backgroundColor: "royalblue", width: "82px" }}
+                                data-testId="navbarlogin"
+                            >
+                                Log In
+                 </Button>
+                        </Link>
+                    }
+                    {(location.pathname == "/login" || location.pathname == "/signup") &&
+                        <Link to={"/"}>
+                            <Button
+                                variant="contained"
+                                color="primary"
+                                style={{ backgroundColor: "royalblue", width: "82px" }}
+                                data-testId="navbarlogin"
+                            >
+                                Home
+                 </Button>
+                        </Link>
+                    }
+
```

Comment:
are you using pretteir? pls check.

Reply:
@Niweera Thank You for the suggestion. I have added commit after modifying with prettier. @Niweera @charithccmc @shehand @rijusougata13 @thisuraseniya I would request you to review.